### PR TITLE
[Cu-370p8vt] Add create database in Makefile

### PR DESCRIPTION
**Pull request by nabhan-au**

### CODE REVIEW

**Comment by sirateek** [comment]
Task linked: [CU-370p8vt Implement "Development database for EC repository."](https://app.clickup.com/t/370p8vt)

File: Makefile [hunk 1]
```diff
@@ -3,4 +3,23 @@ run:
...
-	
\ No newline at end of file
+	
+start-dev-db-linux:
+	docker run --name mysql-ec-dev -d \
+        -p 3306:3306 \
+        -e MYSQL_ROOT_PASSWORD=P@ssw0rd \
+        --restart unless-stopped \
+        -v ${PWD}/database:/docker-entrypoint-initdb.d \
+        mysql:latest
```
Comment (sirateek) [inline Makefile:13]: `docker start mysql-ec-dev  || docker run .....`.
Using this command will start the existed mysql and also create one if not exists

File: Makefile [hunk 2]
```diff
@@ -3,4 +3,23 @@ run:
...
+
+#not try yet
+start-dev-db-window:
+	docker run --name mysql-ec-dev -d \
+        -p 3306:3306 \
+        -e MYSQL_ROOT_PASSWORD=P@ssw0rd \
+        --restart unless-stopped \
+        -v %cd%/database:/docker-entrypoint-initdb.d \
+        mysql:latest
```
Comment (sirateek) [inline Makefile:22]: docker start mysql-ec-dev || docker run ......
Using this command will start the existed mysql and also create one if not exists

File: database/init.sql [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+(
+    ID INTEGER,
+    CitizenID BIGINT,
+    CONSTRAINT fk_citizen_candidate FOREIGN KEY (CitizenID) REFERENCES Population(CitizenID),
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Mp
+(
+    ID INTEGER,
```
Comment (sirateek) [inline database/init.sql:43]: We should add Auto incremental

File: database/init.sql [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    Birthday DATE,
+    Nationality VARCHAR(255),
+    DistrictID INTEGER,
+    CONSTRAINT fk_district FOREIGN KEY (DistrictID) REFERENCES District(DistrictID),
+    PRIMARY KEY (CitizenID)
+);
+
+CREATE TABLE Candidate
+(
+    ID INTEGER,
```
Comment (sirateek) [inline database/init.sql:35]: We should add Auto incremental

**Review by sirateek — APPROVED** [review]

**Review by TaninDean — APPROVED** [review]

**Pull request merged**